server/middleware/verifyToken.js

```diff
@@ -21,7 +21,7 @@ const Auth = {
       if (user.rows[0].length < 0) {
         res.status(401).json({
           status: 401,
-          error: 'Invalid token',
+          error: 'Could not confirm token',
```

Comment:
Should this be `< 0` or `> 0`. Please change the error message to invalid email/password.

Reply:
After finding the `user by id`, if it returns an empty row then the token is invalid i.e. the user does not exist.
Actually, its suppose to be < 1